destination.go

```diff
@@ -29,14 +29,16 @@ import (
 const (
 	// TODO same constant is defined in packages longpoll, logrepl and destination
 	//  use same constant everywhere
```

Comment:
we should be able to get rid of this comment now, maybe @raulb can you do that in your PR plz?

Reply:
@maha-hajja will do!

Reply:
https://github.com/ConduitIO/conduit-connector-postgres/commit/5fe1598dcb2fd8613fdcfb6b0ff0bd61d3e2fc02